src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationAnalysisChartPresenter.cs

```diff
@@ -43,19 +43,18 @@ public override void UpdateAnalysisBasedOn(IAnalysable analysable)
          if (ChartIsBeingUpdated)
          {
             UpdateTemplateFromChart();
-            clearChartAndEditor();
+            ClearChartAndDataRepositories();
          }
          else
             updateCacheColor();
 
-         if (!_parameterIdentification.Results.Any())
-            return;
-
-         _isMultipleRun = _parameterIdentification.Results.Count > 1;
-
-         UpdateAnalysisBasedOn(_parameterIdentification.Results);
+         if (_parameterIdentification.Results.Any())
+         {
+            _isMultipleRun = _parameterIdentification.Results.Count > 1;
+            UpdateAnalysisBasedOn(_parameterIdentification.Results);
+         }
 
-         BindChartToEditors();
+         Refresh();
```

Comment:
This method should also be called from a ChartPresenter in MoBi or PKSim instead of using the ChartUpdater (here we KNOW The presenter)